### DISCUSSION TITLE
Can't submit bug report on homebrew-core

### DISCUSSION THREAD

**Discussion by getaaron** [Homebrew/discussions#2573] [Everyday usage]
The [homebrew-core issues page](https://github.com/Homebrew/homebrew-core/issues/new) is using the new beta template format. I found a reproducible bug and tried to submit a bug report. I get a blank screen and HTTP 400 (bad request) with no helpful error information in the response body. I tried in two separate browsers. Are others able to submit issues? I don't see any recent ones so I'm wondering if we should turn off the beta issue template format.

**Comment by SMillerDev** [comment]
A quick search shows that there are many other people with this issue. It also reveals that the brew team can't fix this:

https://github.com/Homebrew/discussions/discussions/categories/everyday-usage?discussions_q=report+issue+category%3A%22Everyday+usage%22

**Comment by albertz** [comment]
Same for me...

![image](https://user-images.githubusercontent.com/59132/144292628-37bd282f-2c33-4cee-a1e4-04354fb70550.png)

**Reply by alebcay** [reply]
This should be fixed now, please try again.
